torch/_inductor/triton_heuristics.py

```diff
@@ -138,7 +138,9 @@ def launcher({', '.join(def_args)}, grid, stream):
         launcher.n_regs = getattr(binary, "n_regs", None)
         launcher.n_spills = getattr(binary, "n_spills", None)
         launcher.shared = getattr(binary, "shared", None)
-        if config.triton.store_cubin:
+        launcher.store_cubin = config.triton.store_cubin
```

Comment:
Why did this cause a regression if `config.triton.store_cubin` is false by default ?

Reply:
Here it's not causing a problem. It is at Line 236. My theory is accessing the global `config.triton.store_cubin` is expensive when we are running multi-threading (invoking kernels through `.run`).

Reply:
That's very weird, it's happening during compilation only, and shouldn't affect runtime.